bindings/java/src/org/sleuthkit/datamodel/CommunicationsManager.java

```diff
@@ -1409,16 +1379,102 @@ private String normalizePhoneNum(String phoneNum) {
 	 * Utility method to convert a list to an CSV string
 	 */
 	private static <T> String buildCSVString(Collection<T> values) {
-		if (values==null || values.isEmpty()) return "";
+		if (values==null || values.isEmpty()) 
+			return "";
+		
```

Comment:
I think you can use org.apache.commons.lang3.StringUtils.join to make a CSV list instead of rolling your own. See https://stackoverflow.com/questions/15815587/best-way-to-convert-list-to-comma-separated-string-in-java.

Reply:
Are we good with adding a new dependency to the DataModel?  I was trying to avoid that, that's why  I rolled my own.

Reply:
Ok, I forgot I was looking at a SleuthKit PR, not an Autopsy PR.